lib/modules/manager/pip_requirements/artifacts.spec.ts

```diff
@@ -1,17 +1,28 @@
-import { fs } from '../../../../test/util';
+import { fs, mockedFunction } from '../../../../test/util';
 import { GlobalConfig } from '../../../config/global';
+import { exec } from '../../../util/exec';
 import type { UpdateArtifactsConfig } from '../types';
 import { updateArtifacts } from '.';
 
 jest.mock('child_process');
 jest.mock('../../../util/exec');
 jest.mock('../../../util/fs');
+const mockedExec = mockedFunction(exec);
```

Comment:
remove, just call mockExecAll in test and match object after artifacts call. checkout

Reply:
A good sample for using our exec mock

https://github.com/renovatebot/renovate/blob/1111b322f59aae97e5c54a550e5003946cf8bce9/lib/modules/manager/flux/artifacts.spec.ts#L62-L79

Reply:
Thanks. And after reading the implementation of `mockExecAll` I understood what's happening.

In the end, it was most simple to accept the incoming `artifacts.spec.ts` during the merge "as is" and add my extra test cases in fe9d9161380cdf97c85eb6f7294897741e2dc061 again.